modules/cleaning-crio-storage.adoc

```diff
@@ -38,19 +42,19 @@ Follow this process to completely wipe the CRI-O storage and resolve the errors.
 [source,terminal]
 +
 ----
-$ oc adm cordon <nodename>
+$ oc adm cordon <node_name>
```

Comment:
maybe outside the scope of this PR, but I noticed some other instances of `<nodename>` in this module

Reply:
@jldohmann Changes like this are exactly the point of the PR! Thanks!!